pkg/gateway/middleware.go

```diff
@@ -184,6 +184,13 @@ func EnrichWithRepositoryOrFallback(c *catalog.Catalog, authService auth.Gateway
 				fallbackProxy.ServeHTTP(w, req)
 				return
 			}
+
+			// see: github.com/treeverse/lakeFS/issues/3082
```

Comment:
Or just state what this is. If I want to know who added this code, GitHub has excellent blame.